wpilibc/src/main/native/include/frc/event/BooleanEvent.h

```diff
@@ -89,7 +89,7 @@ class BooleanEvent {
    * @param rhs the event to compose with
    * @return the event that is active when both events are active
    */
-  BooleanEvent operator&&(std::function<bool()> rhs);
+  BooleanEvent operator&&(BooleanEvent rhs);
```

Comment:
This is breaking. It should take `function<bool>`, and BooleanEvent's `operator()` should poll the stored value.
It'll capture rhs, but there's not much we can do about that.